features/tags_and_wrangling/tag_wrangling_characters.feature

```diff
@@ -169,3 +169,18 @@ Scenario: character wrangling - syns, mergers, characters, autocompletes
   When I fill in "Synonym of" with "Doctor Who"
     And I press "Save changes"
   Then I should see "Doctor Who is a fandom. Synonyms must belong to the same category."
+
+
+Scenario: Error messages show correct links even if tags contain special characters
```

Comment:
Can you move this scenario to features/tags_and_wrangling/tag_wrangling_special.feature?